jsk_arc2017_baxter/euslisp/lib/baxter-interface.l

```diff
@@ -303,7 +304,7 @@
                                     (send self :get-proximity l/r :left))
                                  (send self :interpolatingp))
                        (unix::usleep 1000))))
-                 (send self :cancel-move-hand)
+                 (send self :cancel-move-hand l/r)
                  (send self :cancel-angle-vector)
```

Comment:
If we want to do that, we have to ` :add-controller` hand controller and initialize `*ri*` with controller including hand controller.
Then we have to change all `:angle-vector` and `:wait-interpolation` to use controllers not including hand controller and ...